charts/kyverno/Chart.yaml

```diff
@@ -79,8 +79,8 @@ annotations:
       description: added TUF flags for custom sigstore deployments
 dependencies:
   - name: grafana
-    version: "0.0.0"
+    version: 3.1.0-rc.7
```

Comment:
I don't think Gradana depends on the kyverno chart. Can we remove this?

Reply:
should I also change the below crd version as well

Reply:
@realshuting hi I am unable to convert the above version only and not modify the version under the dependency section can you help me out

Reply:
Let's keep it as is.